packages/core/src/types/runtime.ts

```diff
@@ -90,23 +90,36 @@ export interface TableDataSourceMeta {
 }
 
 export type DataSourceMeta = TableDataSourceMeta | ImageDataSourceMeta;
-export interface DataAccessor<T = DefaultType> {
+export interface SequentialDataAccessor<T = DefaultType> {
   next: () => Promise<Sample<T> | null>;
   nextBatch: (batchSize: number) => Promise<Array<Sample<T>> | null>;
   seek: (pos: number) => Promise<void>;
 }
+export interface DataAccessor<T = DefaultType> extends SequentialDataAccessor {
+  init: (size: number) => void;
+  nextRandom: () => Promise<Sample<T> | null>;
+  nextBatchRandom: (batchSize: number) => Promise<Array<Sample<T>> | null>;
+  resetRandom: (randomSeed?: string) => Promise<Array<number>>;
```

Comment:
```suggestion
  resetRandom: (randomSeed?: string) => Promise<void>;
```
The script who call the `resetRandom` should not get  the result `Array<number>`.

Reply:
I want `resetRandom` has the same return  values with `createRandom` because developers may use values to do something like variables monitoring.